target/linux/malta/be/config-default

```diff
@@ -1,4 +1,3 @@
 CONFIG_CPU_BIG_ENDIAN=y
-# CONFIG_CPU_LITTLE_ENDIAN is not set
```

Comment:
For malta, this has been used for 11 (!) years. This does not mean it's a good idea, but it certainly breaks your "with 5.4 it was different" argument in the commit message.